cmn/string.go

```diff
@@ -0,0 +1,12 @@
+package cmn
+
+import "strings"
+
+func Sprintf(str ...string) string {
```

Comment:
Could you add comment on how this is different than `fmt.Sprintf`?

Reply:
I have added a [comparison benchmark](https://github.com/NVIDIA/aistore/pull/89/commits/2e1d5a9f3091c7bf71ffacf9d81088d342bb8b95#diff-0251cbcd27a7168f0001d0b5cc9a39c0801bde1f4044780892ce71c02fbe315eR46)
```
// go test -bench="String$" -benchmem  -count=3 .
// BenchmarkTheSprintString-8       2238352               482.8 ns/op          3072 B/op          2 allocs/op
// BenchmarkTheSprintString-8       2614776               472.4 ns/op          3072 B/op          2 allocs/op
// BenchmarkTheSprintString-8       2610601               458.1 ns/op          3072 B/op          2 allocs/op
// BenchmarkPlusString-8             790348              1488 ns/op            9984 B/op          5 allocs/op
// BenchmarkPlusString-8             818689              1373 ns/op            9984 B/op          5 allocs/op
// BenchmarkPlusString-8             811623              1434 ns/op            9984 B/op          5 allocs/op
// BenchmarkSprintfString-8          503763              2257 ns/op           11190 B/op         17 allocs/op
// BenchmarkSprintfString-8          496207              2282 ns/op           11190 B/op         17 allocs/op
// BenchmarkSprintfString-8          474106              2385 ns/op           11190 B/op         17 allocs/op
// PASS
// ok      github.com/NVIDIA/aistore/cmn   12.339s
```